nm/src/nm.c

```diff
@@ -123,8 +123,8 @@ typedef struct {
 #define J4STATUS_NM_FORMAT_APS_SIZE (10+1)
 
 
-#define J4STATUS_NM_DEFAULT_FORMAT_UP_ETH "${addresses}${speed:+(${speed(b)}b/s)}"
-#define J4STATUS_NM_DEFAULT_FORMAT_UP_WIFI "${addresses} (${strength}${strength:+% }${ssid/^.+$/at \\0, }${bitrate(b)}b/s)"
+#define J4STATUS_NM_DEFAULT_FORMAT_UP_ETH "${addresses}${speed:+(${speed(p)}b/s)}"
+#define J4STATUS_NM_DEFAULT_FORMAT_UP_WIFI "${addresses} (${strength}${strength:+% }${ssid/^.+$/at \\0, }${bitrate(p)}b/s)"
```

Comment:
The difference between SI and binary prefixes is not that big, and usually people use binary prefixes but with SI notation. Also, as we can see with your follow-up commit, it breaks the auto-precision of the bitrate (because bitrates are always powers of 2 AFAICT?).
I would love a much bigger commit message explaining how this is better to you.
Also, this is the *default*, you can change it really easily in your own config.

Reply:
I'll update the commit message, but bitrates are rarely a power of two. For example, I want to see that my Ethernet connection is 1Gb/s or 100Mb/s rather than 953.674316Mib/s. Similar for WiFi, there are bitrates like 144.4Mb/s, 54Mb/s etc. With binary prefix, the former is 137.710571Mib/s.

Reply:
I see I used SI prefixes in my nl plugin (though it doesn’t have Ethernet link speed, only WiFi bitrates). So let’s go for it. Please squash the `(p.0)` directly for now. (I may improve the auto-precision a bit in the future though.)

Reply:
By "squash" you mean to squash the two commits to a single one or to not include it in the commit?